.github/workflows/deps.yml

```diff
@@ -0,0 +1,45 @@
+name: Dependency Changes
+
+# Trigger on PRs.
+on:
+  pull_request:
+
+permissions:
+  contents: read
+
+jobs:
+  # Compare dependencies before and after this PR.
+  dependencies:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    strategy:
+      fail-fast: true
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: stable
```

Comment:
Should we explicitly specify the version instead?

Reply:
This is zero-maintenance, which is why I did it this way.  I think it's fine since it uses the same version before/after.

Reply:
Ack.